src/pages/instances/InstanceOverviewDeviceDetail.tsx

```diff
@@ -26,15 +26,13 @@ const InstanceOverviewDeviceDetail: FC<Props> = ({ device, project }) => {
 
     return (
       <>
-        <span>
-          volume&nbsp;
-          <ResourceLink
-            type={"volume"}
-            value={device.source as string}
-            to={`/ui/project/${project}/storage/pool/${device.pool}/volumes/custom/${device.source}`}
-          />{" "}
-        </span>
-        <span> on pool </span>
+        Volume&nbsp;
```

Comment:
for some reason it seems like that `&nbsp;` isn't working on really small screen sizes?

![Uploading Screenshot from 2024-10-23 21-34-52.png…]()

Reply:
Replaced it with a regular space as there is no good solution if we run out of space having them in one line would hide the content, which is even worse than splitting by lines.